storage/postgres/testdb/testdb.go

```diff
@@ -120,3 +120,13 @@ func sanitize(script string) string {
 func getConnStr(name string) string {
 	return fmt.Sprintf("database=%s %s", name, *pgOpts)
 }
+
+// OpenTestDBOrDie attempts to return a connection to a new postgres
+// test database and fails if unable to do so
+func OpenTestDBOrDie() *sql.DB {
+	db, err := NewTrillianDB(context.TODO())
```

Comment:
Any reason not to pass a `context` in here rather than adding a TODO? (Does this have to fit with an existing interface?)

Reply:
No reason, I was mostly copying things over from the mysql tests. I'll add a context with a 30 second timeout here